packages/core/src/rules/common/response-contains-header.ts

```diff
@@ -0,0 +1,25 @@
+import { Oas2Rule, Oas3Rule } from '../../visitors';
+import { UserContext } from '../../walk';
+import { Oas3Response } from '../../typings/openapi';
+import { Oas2Response } from '../../typings/swagger';
+
+export const ResponseContainsHeader: Oas3Rule | Oas2Rule = (options) => {
+  const mustExist = options.mustExist || [];
+  return {
+    Response: {
+      skip: (_response: Oas2Response | Oas3Response, key: any) => {
+        return !['200', '201', '202'].includes(key.toString());
```

Comment:
So this rule checks only 200 201 and 202? What if I want to check other responses headers?

Maybe we should change option to look like this:

```yaml
response-contains-header:
  severity: error
  names:
    2xx: [ 'x-request-id' ]
    400: [ 'Content-Type' ]
```

Reply:
Yes, this would be great.